File: src/RepositoryModules.CS/IOHelper.cs
using System.Drawing;
using System.Drawing.Imaging;
using System.IO;

namespace RepositoryModules
{
    static class IOHelper
    {
        public static void WriteImage(string path, byte[] image)
        {
            using (var ms = new MemoryStream(image))
            using (var bitmap = new Bitmap(ms))
            {
                bitmap.Save(path, ImageFormat.Jpeg);
            }
        }
    }
}

File: src/RepositoryModules.VB/IOHelper.vb
Imports System.Drawing.Imaging
Imports System.IO

Friend Module IOHelper

    Public Sub WriteImage(Path As String, Image As Byte())

        Using MS = New MemoryStream(Image)

            Using BM = New Bitmap(MS)

                BM.Save(Path, ImageFormat.Jpeg)

            End Using
        End Using
    End Sub
End Module
